crowbar_framework/app/models/ceilometer_service.rb

```diff
@@ -33,9 +33,7 @@ def role_constraints
         "ceilometer-agent" => {
           "unique" => false,
           "count" => -1,
-          "exclude_platform" => {
-            "windows" => "/.*/"
-          }
+          "exclude_platform" => {}
```

Comment:
So go ahead and add it to ALL barclamps and roles please. If you don't want to do that remove it :)